### ISSUE TITLE
IDs used in ARIA and labels must be unique

### ISSUE THREAD

**Issue by christinach** [pulibrary/lib_jobs#538]
### Steps to recreate
Run the accessibility spec

### URL
[spec/system/accessibility_spec.rb](https://github.com/pulibrary/lib_jobs/tree/main/spec/system/accessibility_spec.rb)

### Issues
```
1) duplicate-id-aria: IDs used in ARIA and labels must be unique (critical)
           https://dequeuniversity.com/rules/axe/4.7/duplicate-id-aria?application=axeAPI
           The following 1 node violate this rule:
           
               Selector: .full-logo > title
               HTML: <title data-v-1117c808="" id="logo-library">Princeton University Library Logo</title>
               Fix any of the following:
               - Document has multiple elements referenced with ARIA with the same id attribute: logo-library
 ```

### Severity
critical

### WCAG Conformance level (A, AA, AAA)
WCAG 2.1 (A)
WCAG 2.0 (A)

### User impact
Disabilities Affected:
Blind
Deafblind

### Recommended fix
Update [lux](https://github.com/pulibrary/lux) to the latest version

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

### Notes

**Comment by sandbergja** [comment]
My PR didn't actually fix this, re-opening
